express_relay/sdk/python/express_relay/searcher/examples/simple_searcher_svm.py

```diff
@@ -189,17 +240,28 @@ async def main():
     log_handler.setFormatter(formatter)
     logger.addHandler(log_handler)
 
+    if args.private_key:
```

Comment:
what if both empty?

Reply:
since the mutually exclusive group required flag is True, this should not happen